src/controllers/expensesController.js

```diff
@@ -0,0 +1,40 @@
+const Expense = require('./Expense');
+const { mockExpenses } = require('./data');
+
+const getExpenses = () => [...mockExpenses.values()];
+
+const getExpenseById = (id) => mockExpenses.get(id);
+
+const addExpense = (userId, spentAt, title, amount, category, note) => {
+  const maxId = Math.max(...mockExpenses.keys(), 0);
+  const newId = maxId + 1;
```

Comment:
You can merge both lines into one:
`const newId = Math.max(...mockExpenses.keys(), 0) + 1;`